src/components/SearchSelectInput/EmptyOptions/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import styles from './styles.css';
+
+interface DefaultEmptyComponentProps {
+    optionsPending?: boolean;
+    isFiltered?: boolean;
+}
+
+function EmptyOptions(props: DefaultEmptyComponentProps) {
+    const {
+        isFiltered = false,
+        optionsPending = false,
+    } = props;
+
+    if (optionsPending) {
+        // FIXME: use loading
```

Comment:
@frozenhelium , lets create a new issue in zube for using loading animation here